src/makeCalculator.js

```diff
@@ -4,7 +4,31 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  let result = 0;
+
+  return {
+    add: (num) => (result += num),
+
+    subtract: (num) => (result -= num),
+
+    multiply: (num) => (result *= num),
+
+    divide: (num) => (result /= num),
+
+    operate: function (callback, num) {
```

Comment:
You can use the method shorthand syntax for defining methods within an object. Instead of using 'function', you can define 'operate' as 'operate(callback, num) { ... }'. This aligns with the checklist item: 'if you creating a method in the object, you don't need to use function keyword, use shortcut instead.'